erpnext_documentation/www/docs/user/manual/en/setting-up/delete-company-transactions.md

```diff
@@ -0,0 +1,45 @@
+<!-- add-breadcrumbs -->
+# Delete Company Transactions
+
+**You can delete all the records associated with a company and start afresh, while keeping the other master data like Customers, Items, BOMs intact.**
```

Comment:
"You can delete all the **transaction data like sales invoices, sales orders** associated with a company and start afresh, while keeping the other master data like **c**ustomers, **i**tems, BOMs intact."

Better to say 'transaction data' instead of 'all the records'.

Reply:
Done.